src/integrationTest/java/uk/gov/hmcts/reform/ccd/config/WireMockStubs.java

```diff
@@ -21,10 +28,18 @@ public class WireMockStubs {
 
     private static final String DOCUMENTS_DELETE_PATH = "/documents/delete";
     private static final String ROLES_DELETE_PATH = "/am/role-assignments/query/delete";
+    private static final String ROLES_QUERY_PATH = "/am/role-assignments/query";
+
+    private RoleAssignmentsResponse roleAssignmentsResponse = new RoleAssignmentsResponse();
 
     public void setUpStubs(final WireMockServer wireMockServer) {
+        QueryResponse queryResponse = new QueryResponse();
```

Comment:
I'm not sure that you need the following snippet at all.


 QueryResponse queryResponse = new QueryResponse();
        queryResponse.setId(UUID.randomUUID());
        roleAssignmentsResponse.setRoleAssignmentResponse(Collections.singletonList(queryResponse));